app/assets/javascripts/discourse/tests/acceptance/user-test.js

```diff
@@ -131,3 +136,46 @@ acceptance("User Routes - Moderator viewing warnings", function (needs) {
     assert.ok($("div.alert-info").length, "has the permissions alert");
   });
 });
+
+acceptance("User - Saving user options", function (needs) {
+  needs.user({
+    admin: false,
+    moderator: false,
+    username: "eviltrout",
+    id: 1,
+    user_option: EmberObject.create({}),
+  });
+
+  needs.settings({
+    disable_mailing_list_mode: false,
+  });
+
+  test("saving user options", async function (assert) {
+    pretender.put("/u/eviltrout.json", () => {
+      return [
+        200,
+        { "Content-Type": "application/json" },
+        { success: true, user: {} },
+      ];
+    });
```

Comment:
Is there a reason we didn't use `needs.pretender` here to setup the pretender?

Reply:
Ah good point, didn't have that in the unit test, will change :)